src/postgraphql/cli.ts

```diff
@@ -117,7 +116,9 @@ const server = createServer(postgraphql(pgConfig, schemas, {
   graphiql: !disableGraphiql,
   jwtPgTypeIdentifier,
   jwtSecret: jwtSecret || deprecatedJwtSecret,
-  jwtOptions,
+  jwtOptions: {
+    audience: jwtAudience,
```

Comment:
Could we flatten this out for now into just `jwtAudience` and not `jwtOptions: { audience }`? In the future we could put all of the jwt settings (for which there are more then a few couple…) in a `jwtOptions` property. It also removes the need for the type.